part-one-answer.html

```diff
@@ -0,0 +1,28 @@
+<html>
```

Comment:
This filename tho :trollface:

Reply:
That was, honestly, a copy paste error... found a how to and then made it to my liking... for everything except the filename, apparently.

Reply:
We call that a Copy Pasta. Gets you every time 😄